Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -446,6 +449,25 @@ public class JavaScriptLifter: ComponentBase, Lifter {
                 
             case let op as Comment:
                 w.emitComment(op.content)
+
+            case is BeginCodeString:
+                if nestedCodeString > 0 {
+                    w.emit("\(constDecl) \(instr.output) = `")
+                } else {
+                    w.emit("\(constDecl) \(instr.output) = \\`")
```

Comment:
This will only work for one nesting level though, not multiple ones, will it?

Reply:
Incrementing and decrementing the nesting counter when it encounters Being and End CodeStrings should handle the following cases:
```
` outer block code
  ` inner code block1
   ` inner code block 2
   `
 `
  outer block code
`
```
and cases where you have:
```
` outer code block
  ` code block 1
  `
  ` code block 2
  `
  outer code block
`
```

I'm not sure if there's a case that I've missed here

Reply:
oh, i see, I've got the order wrong :(

Reply:
Hm I can't seem to reproduce that, it gives me an invalid syntax exception. Are you sure escaping this way is possible?
Also I think in any case it would be good to add a test for this to https://github.com/googleprojectzero/fuzzilli/blob/master/Tests/FuzzilliTests/LifterTest.swift that builds up at least three levels of codestring nesting, lifts the resulting program, then compares the output to a constant string.

Reply:
yes, you're probably seeing syntax errors since the first back tick was being incorrectly lifted to `\\`` which was a bug.
I'll also add a test case which is a good idea.

Reply:
I've fixed the escaping bug add added testcases to verify lifting